### DISCUSSION TITLE
Editor Team Question: Performance

### DISCUSSION THREAD

**Discussion by minecraftchloe** [Mojang/minecraft-editor#3] [Q&A]
Please let us know with as much detail as possible the conditions under which you crashed the Editor or made it stall out. 

Using blocks that are subject to gravity is one area that we know we need to improve on, as well as the general transaction size for multiblock placement, but what else have you found? 

Helpful details: 
- Preview build version
- Video or screenshots 
- Local, peer-to-peer, dedicated server and how many creators
- Repro steps
- Graphics card (eg. DX11 or DX12 - [how to find your DirectX version](https://support.microsoft.com/en-us/windows/checking-your-version-of-directx-7b71e74f-02e8-456f-72c7-9a1c1bbf0e9a))
- Device info (eg. desktop HP Omen, laptop Acer Nitro)

**Comment by JWForever5504** [comment]
I caused Editor Mode to get locked in crosshair mode (unable to do anything but force close) after trying to paste an entire mountain range. So maybe I was trying to crash it...

I was able to copy just fine, with it also saying "Selection Copied." However, it didn't paste a single block, and when I went into crosshair mode, I couldn't go back into tool mode.

Details I have:
Graphics Card - NVIDIA GeForce RTX 2070
Device Info - Windows 10 Desktop (10.0.19044.2604???) I don't really know this unfortunately.
Local World with 1 Player
No videos/screenshots
Version - 1.19.80.20
